.github/labeler.yml

```diff
@@ -609,7 +609,7 @@ Numbers:
   - any-glob-to-any-file: 'tests/unit/ingredients_nutriscore.t'
   - any-glob-to-any-file: 'tests/unit/ingredients_nutriscore.t'
   - any-glob-to-any-file: 'scripts/add_nutriscore_to_scanbot_csv.pl'
-  - any-glob-to-any-file: 'docs/introduction/get-the-nutriscore.md'
+  - any-glob-to-any-file: 'docs/api/api-tutorials/get-the-nutriscore.md'
```

Comment:
I think this page is more an How-to than a Tutorial (see https://diataxis.fr/tutorials-how-to/#tutorials-how-to )